src/common-utils/index.js

```diff
@@ -71,9 +75,7 @@ module.exports = {
     },
 
     // Returns if a value is a string
-    isString(value) {
-        return typeof value === 'string' || value instanceof String;
-    },
+    isString,
 
```

Comment:
I would suggest this fix recommended by @midrissi to bring the isString in scope

https://github.com/pajaydev/ebay-node-api/blob/master/src/common-utils/index.js#L68-L71

```
upperCase(data) {
  if (!this.isString(data)) data = data.toString();
  return data.toUpperCase();
},
```

Reply:
I tried it and it gave me ""this" is not defined""   there is actually no context "this" in that environment. 
However if you wanna bring the isString() in scope at all cost i'll see another alternative, i'm also open for your suggestions guys.

Reply:
Not working TypeError: Cannot read property 'isString' of undefined

Reply:
I think we need to bind the this while calling like this 
```
upperCase.call(this, marketPlaceId);
```

But I feel this is fine.

Reply:
`this` in that scope refers to the ebay object however i did this and it worked : 
`const Utils = require('./common-utils');`
`    marketPlaceId = upperCase.call(Utils, marketPlaceId);
`

Reply:
@pajaydev   Anyway thank you so much for mergeing my PR you really made my day :))))

Reply:
Cool 👍